CONTRIBUTING.md

```diff
@@ -121,10 +121,10 @@ If you have not installed nix follow the instructions [here](https://nixos.org/)
 
 You can configure the project [cachix](https://cachix.org) binary cache with this command: `nix shell nixpkgs#cachix --command cachix use change-metrics`.
 
-You can also use the cachix store directly, for example to run the monocle command:
+You can also use the cachix binary store directly, for example to run the latest release:
 
 ```ShellSession
-nix run . --option binary-caches "https://cache.nixos.org https://change-metrics.cachix.org" --option trusted-public-keys "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= change-metrics.cachix.org-1:dCe8jx9vptiF6DCdZ5y2QouvDsxgFRZnbHowhPnS4C0="
+nix run github:change-metrics/monocle/1.7.0 --option binary-caches "https://cache.nixos.org https://change-metrics.cachix.org" --option trusted-public-keys "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= change-metrics.cachix.org-1:dCe8jx9vptiF6DCdZ5y2QouvDsxgFRZnbHowhPnS4C0="
```

Comment:
No way to simply specify master ?
1.7.0 is not the latest release. Latest release is 1.6.0 as stated in the README.md.
I feel it create confusion.

Reply:
Well yes, it's just that we only cache the final binary for release. Running this for master is likely to cause the full toolchain to be downloaded, while using a pin should reduce the download to only the command, which should be about 20MB.